### ISSUE TITLE
check_vmware_alarms | logger includes fields not used by this plugin

### ISSUE THREAD

**Issue by atc0005** [atc0005/check-vmware#258]
https://github.com/atc0005/check-vmware/blob/e272cf227c097ca2174c212043f306913f4706b9/cmd/check_vmware_alarms/main.go#L84-L89

I optimistically (and incorrectly) assumed that I knew how I would end up implementing support for filtering based on VM name and resource pools. Instead, support was added for filtering based on *object* (managed entity) name (not specific to VMs) and (at least for now) a set of separate config settings for lists of resource pools.

In short, all three additional structured logging fields are incorrect and should be removed. Since there are so many filtering options currently available, it is probably best to skip adding replacement fields to the "base" logger used by this plugin.
